### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by SirSevenG**
```python
            try:
                lowest_ask = min([float(x['price']) for x in pair_orderbook["asks"]])
            except ValueError:
                lowest_ask = float("{:.2f}".format(0.0))
            try:
                highest_bid = max([float(x['price']) for x in pair_orderbook["bids"]])
            except ValueError:
                highest_bid = float("{:.2f}".format(0.0))
```
Not 100% sure for above

### CODE REVIEW

File: data_fetcher.py [hunk 1]
```diff
@@ -1,18 +1,20 @@
...
+
+
+# adex_tickers = ["AWC", "AXE", "BAT", "BCH", "BET", "BOTS", "BTC", "BUSD", "CCL", "CHIPS", "CRYPTO", "DAI", "DASH",
+#                 "DEX", "DGB", "DOGE", "ECA", "EMC2", "ETH", "FTC", "HUSH", "ILN", "JUMBLR", "KMD", "LABS", "LTC",
+#                 "MCL", "MGW", "MORTY", "NAV", "OOT", "PANGEA", "PAX", "QTUM", "REVS", "RFOX", "RICK", "RVN",
+#                 "SUPERNET", "TUSD", "USDC", "VRSC", "XZC", "ZEC", "ZER"]
+#
+# 45 tickers atm = 1980 pairs // 283 in db as of 09.27
+# possible_pairs = list(itertools.permutations(adex_tickers, 2))
+possible_pairs = get_pairs()  # ~25s execution vs ~207s for above
```
Comment (tonymorony) [inline data_fetcher.py:17]: where can I find get_pairs function?

Comment (SirSevenG) [reply data_fetcher.py:17]: utils/util_funct.py

Comment (SirSevenG) [reply data_fetcher.py:17]: ^^ 
`from utils.util_funct import find_unique_pairs as get_pairs`

Comment (tonymorony) [reply data_fetcher.py:17]: oh lol, not noticed imports

**Comment by dth88** [comment]
ok, I didn't expect that you PR directly into my feature branch, I'm still working on the parser and will push the final updates later today, lets merge them a bit later, please.

**Pull request merged**